docs/source/design.rst

```diff
@@ -0,0 +1,3 @@
+How it works
```

Comment:
Yeah. I think we also need a broader discussion about what should and shouldn't be included here, since we already have a website.